{
  "name": "throws-exception",
  "version": "0.0.2",
  "description": "Function that returns a boolean indicating whether an exception was thrown or not",
  "repository": {
    "type": "git",
    "url": "https://github.com/ProperlyExplained/throws-exception"
  },
  "main": "index.js",
  "scripts": {
    "test": "echo \"TODO: Add tests\" && exit 0"
  },
  "keywords": [
    "check",
    "throw",
    "throws",
    "exception",
    "if",
    "boolean",
    "try",
    "catch"
  ],
  "author": "Adrian Cuadrado",
  "license": "ISC"
}
